src/pages/track-menu.f7.html

```diff
@@ -19,17 +19,18 @@
             <div class="menu-grid">
                 <a href="/wellness-quiz/" class="menu-grid__item">
```

Comment:
```suggestion
                <a href="#" class="menu-grid__item">
```

Se você clica aqui, mesmo estando no grayscale ele direciona pra notfound. Se deixar sem o link fica melhor.